applications/EigenSolversApplication/CMakeLists.txt

```diff
@@ -16,10 +16,6 @@ message( "**** configuring KratosEigenSolversApplication ****" )
 ################### PYBIND11
 include(pybind11Tools)
 
-if( NOT DEFINED EIGEN_ROOT )
```

Comment:
As the EIGEN_ROOT variable is not needed anymore, the README of the application has to be adapted.

Reply:
done